en/docs/learn/account-disabling.md

```diff
@@ -33,8 +33,8 @@ locking and disabling. Follow the below steps to do this configuration.
         This will remove the account disabling option from all tenants.
 
         ``` java
```

Comment:
The protocol should be toml—correct in other relevant places.

Reply:
modified